docs/deploy-custom-image.rst

```diff
@@ -4,90 +4,114 @@
 Deploying custom Docker images
 ==============================
 
-Scrapy Cloud 2.0 allows you to run your spiders in custom Docker containers. The tools for managing and deploying Docker images are bundled under ``shub image``.
+.. note::
 
-Deployment
-==========
-This section describes how to build and deploy to Scrapy Cloud 2.0 a custom Docker image for a Scrapy project.
-
-.. _step-one:
+    This feature is currently only available for paying customers.
 
-1. Initialization
------------------
-Open up a terminal and go to your crawler's project folder in your local machine::
+It's possible to deploy Docker images with spiders to Scrapy Cloud. To be able to run spiders in custom `Docker`_
+images it's necessary to follow the :ref:`Custom images contract <custom-images-contract>` - a set of requirements
+that image should comply with to be compatible with Scrapy Cloud. The tools for managing and deploying
+Docker images are bundled under ``shub image``.
 
-    $ cd path/to/your/project
+.. _Docker: https://docs.docker.com/
 
-And then run the :ref:`init command <commands-init>`::
-
-    $ shub image init --requirements path/to/requirements.txt
+Deployment
+==========
 
-This will create a Dockerfile for your container including ``requirements.txt`` as a dependency and using `python:2.7 <https://hub.docker.com/r/library/python/>`_ as `the base image <https://docs.docker.com/engine/reference/builder/>`_ for your custom image. If you want to use a different one, pass the ``--base-image`` option, like this::
+This section describes how to build and deploy a custom Docker image to Scrapy Cloud. For all the following steps
+it's assumed that commands are executed at the root directory of your project.
 
-    $ shub image init --base-image scrapinghub/base:12.04
 
-In this case, it will use the image available at https://hub.docker.com/r/scrapinghub/base tagged with ``12.04``.
+1. Add scrapinghub.yml
+----------------------
```

Comment:
I agree that we can remove this section. Since @dangra suggested that we drop support for custom repositories completely at some point, I wonder if we should simply not mention it in the docs at all?

The https://shub.readthedocs.io/en/stable/configuration.html needs some love, it would be good to have a "full-fledged" configuration in there and have a list of all available options. I can prepare a PR for that tomorrow